operations/observability/mixins/meta/rules/server.yaml

```diff
@@ -46,13 +46,12 @@ spec:
 
     # Rollout alerts
     - alert: JsonRpcApiErrorRates
-      # Reasoning: the values are taken from past data
       expr: sum(rate(gitpod_server_api_calls_total{statusCode!~"2.*|3.*|4.*"}[5m])) by (cluster) / sum(rate(gitpod_server_api_calls_total[5m])) by (cluster) > 0.04
```

Comment:
[nit] We could flip the `statusCode` expression to `5.*` for slightly more readability